### PR TITLE
Add kickstart files for RHEL7 and RHEL8 ANSSI BP28 High profile.

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Add kickstart files for RHEL7 and RHEL8 ANSSI BP28 High profile.

### CODE REVIEW

File: rhel8/kickstart/ssg-rhel8-anssi_bp28_high-ks.cfg [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+clearpart --linux --initlabel
+
+# Create primary system partitions (required for installs)
+part /boot --fstype=xfs --size=512
+part pv.01 --grow --size=1
+
+# Create a Logical Volume Management (LVM) group (optional)
+volgroup VolGroup --pesize=4096 pv.01
+
+# Create particular logical volumes (optional)
```
Comment (yuumasato) [inline rhel8/kickstart/ssg-rhel8-anssi_bp28_high-ks.cfg:111]: Based on https://github.com/ComplianceAsCode/content/pull/6340
Can you add partitions for `/usr`, `/opt` and `/srv`?

**Review by yuumasato — CHANGES_REQUESTED** [review]
A few partitions required by ANSSI are missing in both kickstarts.

File: rhel8/kickstart/ssg-rhel8-anssi_bp28_high-ks.cfg (hunk 1 above)
Comment (ggbecker) [reply rhel8/kickstart/ssg-rhel8-anssi_bp28_high-ks.cfg:111]: Yes, I can. I thought the ones from OSPP should cover everything already. I'll include them.

Comment (yuumasato) [reply rhel8/kickstart/ssg-rhel8-anssi_bp28_high-ks.cfg:111]: The sizing of the disk and/or other partitions may need to be changed too.

Comment (ggbecker) [reply rhel8/kickstart/ssg-rhel8-anssi_bp28_high-ks.cfg:111]: please check: https://github.com/ComplianceAsCode/content/pull/6482/commits/9de6320142ffc456e77451413879fdc188657396 I don't if the `fsoptions` are correct. Can you confirm?

Comment (yuumasato) [reply rhel8/kickstart/ssg-rhel8-anssi_bp28_high-ks.cfg:111]: From ANSSI R12, `/opt` and `/srv` should have `"nosuid,nodev"`.

Comment (ggbecker) [reply rhel8/kickstart/ssg-rhel8-anssi_bp28_high-ks.cfg:111]: ok, thanks.

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Review by yuumasato — APPROVED** [review]

**Pull request merged**